pgoapi/pgoapi.py

```diff
@@ -380,8 +384,33 @@ def cleanup_pokemon(self, inventory_items=None):
             if len(pokemons) > MIN_SIMILAR_POKEMON:
                 pokemons = sorted(pokemons, key=lambda x: (x.iv, x.cp), reverse=True)
                 # keep the first pokemon....
+                last_pokemon = pokemons[0]
                 for pokemon in pokemons[MIN_SIMILAR_POKEMON:]:
```

Comment:
I believe at the time of this PR MIN_SIMILAR_POKEMON was still a hard coded value and not changeable through the config. The line in question wasn't even part of my change set